src/api.c

```diff
@@ -14,14 +14,18 @@ API newAPI() {
 #ifdef __x86_64
   api.init = &initVirtual;
   api.getTemperature = &getTemperatureVirtual;
-  api.getHeaterStatus = &getHeaterStatusVirtual;
+  api.getActualHeaterStatus = &getHeaterStatusVirtual;
+  api.getHeaterVariable = &getHeaterStatusVirtual;
+  api.getSBHeaterStatus = &getHeaterStatusVirtual;
```

Comment:
I think these identifiers are not descriptive and pretty confusing.  
e.g., what's a `HeaterVariable`? 

Please give them more descriptive names. For example, `getHeaterPowerStatus` and `getHeaterReadyStatus` describe clearly what they return.

Reply:
Also, what is the purpose of having both `api.getHeaterVariable`and  `api.getSBHeaterStatus` when they point to the same function `&getHeaterStatusVirtual`?